src/app/components/HoldRequest/HoldRequest.jsx

```diff
@@ -36,9 +36,8 @@ class HoldRequest extends React.Component {
   }
```

Comment:
Might not be related here. But why we need `return false` and `return true` in this function?